packages/slate-types/src/nodes/ContactNode.ts

```diff
@@ -7,8 +7,9 @@ export const CONTACT_NODE_TYPE = 'contact';
 
 export interface ContactNode extends ElementNode {
     type: typeof CONTACT_NODE_TYPE;
-    contact: PressContact;
     uuid: string;
+    reference: ContactNode['uuid'] | null;
```

Comment:
Isn't it better to do `string | null` instead of the type referencing itself? 🤔 `uuid` and `reference` are supposed to be independent of each other anyway, right?

Reply:
Oh, good catch! 

So it actually had to be `reference: NewsroomContact['uuid']`. 

Fixed with be2421afdcc387436960b391c9b901bd5f3441f3